include/swift/AST/Import.h

```diff
@@ -85,7 +85,10 @@ enum class ImportFlags {
   Preconcurrency = 0x20,
 
   /// Used for DenseMap.
-  Reserved = 0x80
+  Reserved = 0x80,
+
+  /// Silence all deprecation warnings coming from symbols imported from the module
+  IgnoresDeprecationWarnings = 0x90
```

Comment:
I would keep `Reserved` as the last case, and insert `IgnoresDeprecationWarnings` before it. Note also that this enum is used as an option set, meaning that each case needs to correspond to a value that has a single unique bit in binary, e.g `001`, `010`, `100` etc. These are in hex, so the next one after `0x80` would be `0x100`. However it looks like there's already a gap between `Preconcurrency` and `Reserved`, so you can insert `IgnoresDeprecationWarnings` between them as `0x40`.

Reply:
There